plugins/mcreator-localization/help/default/biome/coverage_estimate.md

```diff
@@ -0,0 +1,9 @@
+This field provides an estimate of how much of the overworld surface would be 
+covered by this biome. Only works properly if values for biomes are between -1 and 1 as this
+is the range overworld biomes use.
+
+In case your biome will be used in nether or in overworld caves, this estimation
+is not valid.
+
+When using biomes in custom dimensions, coverage depends on the amount of biomes
+your dimension has, and biome generation parameters of biomes in the dimension.
```

Comment:
Maybe make this shorter a little bit (also fixes an unknown <kbd>if will, then is</kbd> conditional in first sentence)?
```suggestion
This estimation is not valid in case your biome is used in Nether, Overworld caves
or custom dimensions.
```

Reply:
Then this important piece of information is left out:

> coverage depends on the amount of biomes your dimension has, and biome generation parameters of biomes in the dimension.

Reply:
OKay, but still, the conditional used here and dimension names should be fixed:
```suggestion
In case your biome is used in Nether or in Overworld caves, this estimation
is not valid.

When using biomes in custom dimensions, coverage depends on the amount of biomes
your dimension has, and biome generation parameters of biomes in the dimension.
```